tests/devices/qubit/test_measure.py

```diff
@@ -96,6 +97,49 @@ def test_sum_sum_of_terms_when_backprop(self):
         state = qml.numpy.zeros(2)
         assert get_measurement_function(qml.expval(S), state) is sum_of_terms_method
 
+    @pytest.mark.usefixtures("use_legacy_opmath")
+    def test_hamiltonian_with_multi_wire_obs(self):
+        """Check that a Hamiltonian with a multi-wire observable uses the sum of terms method."""
+
+        S = qml.Hamiltonian(
+            [0.5, 0.5],
+            [
+                qml.X(0),
+                qml.Hermitian(
+                    np.array(
+                        [
+                            [0.5, 1.0j, 0.0, -3j],
+                            [-1.0j, -1.1, 0.0, -0.1],
+                            [0.0, 0.0, -0.9, 12.0],
+                            [3j, -0.1, 12.0, 0.0],
+                        ]
+                    ),
+                    wires=[0, 1],
+                ),
+            ],
+        )
+        state = np.zeros(2)
+        assert get_measurement_function(qml.expval(S), state) is sum_of_terms_method
+
+    def test_no_sparse_matrix(self):
+        """Tests that Hamiltonians/Sums containing observables that does not have sparse matrix."""
```

Comment:
```suggestion
        """Tests Hamiltonians/Sums containing observables that do not have a sparse matrix."""
```

Is this the meaning of the statement? Or am I missing something?